src/components/skDashboard/SocialMediaCard.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { StyledCard, StyledFollowerCount, StyledIcon, StyledNamesText } from './styles/SocialMediaCard.styled'
+type Props = {
```

Comment:
leave a line between imports and the rest of the code

Reply:
a95d557